Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectDetails.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+import Kingfisher
+import KsApi
+import SwiftUI
+
+struct PPOProjectDetails: View {
+  let imageUrl: URL?
+  let title: String?
+  let pledge: GraphAPI.MoneyFragment
+
+  var body: some View {
+    HStack {
+      KFImage(self.imageUrl)
+        .resizable()
+        .clipShape(Constants.imageShape)
+        .aspectRatio(Constants.imageAspectRatio, contentMode: .fit)
+        .frame(width: Constants.imageWidth)
+      Spacer()
+        .frame(width: Constants.spacing)
+      VStack {
+        if let title {
+          Text(title)
+            .font(Font(Constants.titleFont))
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .lineLimit(2)
+        }
+        // TODO: Localize
+        Text("\(self.pledge.symbol ?? "")\(self.pledge.amount ?? "") pledged")
+          .font(Font(Constants.subtitleFont))
+          .foregroundStyle(Color(Constants.subtitleTextColor))
+          .frame(maxWidth: .infinity, alignment: .leading)
+          .lineLimit(1)
+      }
+    }
+    .fixedSize(horizontal: false, vertical: true)
+    .frame(maxWidth: .infinity)
+  }
+
+  private enum Constants {
+    static let imageShape = RoundedRectangle(cornerRadius: 4)
+    static let imageAspectRatio: CGFloat = 16 / 9
+    static let imageWidth: CGFloat = 85
+    static let spacing: CGFloat = 8
```

Comment:
nit: Not super important if it'll lead to a lot of extra work, but I think we should try to stick to using `Styles.grid()` and `Styles.gridHalf()` for spacing, just so that the app looks cohesive.